doc/man3/flux_msg_create.rst

```diff
@@ -0,0 +1,68 @@
+==========================
+flux_msg_create(3)
+==========================
+
+.. default-domain:: c
+
+SYNOPSIS
+========
+
+.. code-block:: c
+
+   #include <flux/core.h>
+
+   flux_msg_t *flux_msg_create (int type)
+
+   flux_msg_t *flux_msg_copy (const flux_msg_t *msg, bool payload)
+
+   const flux_msg_t *flux_msg_incref (const flux_msg_t *const_msg)
+
+   void flux_msg_decref (const flux_msg_t *const_msg)
+
+   void flux_msg_destroy (flux_msg_t *msg)
+
+
+DESCRIPTION
+===========
+
+:func:`flux_msg_create` creates a `flux_msg_t` of `int type`. Different types
+of Flux messages are defined `RFC 6 <https://flux-framework.readthedocs.io/
+projects/flux-rfc/en/latest/spec_6.html>_`. All messages have a starting reference
+count of 1.
+
+:func:`flux_msg_copy` copies the attributes of a `flux_msg_t` object into a new
+`flux_msg_t` object.
+
+:func:`flux_msg_incref` provides a constant pointer to a `flux_msg_t` object.
+It also increments the reference count of the object by 1.
+
+:func:`flux_msg_decref` decrements the reference count of a `flux_msg_t` object
+by 1. When the reference count reaches 0, the message is destroyed.
+
+:func:`flux_msg_destroy` destroys a message.
+
+RETURN VALUE
+============
+
+:func:`flux_msg_create` returns a `flux_msg_t` type.
+
+:func:`flux_msg_copy` returns a new `flux_msg_t` object with the same attributes
+as another `flux_msg_t` objects.
+
+:func:`flux_msg_incref` returns a constant pointer to a `flux_msg_t` object.
```

Comment:
This section doesn't need to explicitly call out the type.  But it should tell you what the return possiblities are.  For example, on failure, create and copy return a Flux message on success, and on failure, return NULL and set errno.  Incref just returns its :var:`msg` argument for convenience.

Reply:
I think I addressed this one, by adding an `ERRORS` section and mentioning the different errors that could be raised by each in the `RETURN VALUE` section. But let me know if it's redundant to have it mentioned in both places, or needs to be done differently.